go/processor/test_utils.go

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2024 Fantom Foundation
+//
+// Use of this software is governed by the Business Source License included
+// in the LICENSE file and at fantom.foundation/bsl11.
+//
+// Change Date: 2028-4-16
+//
+// On the date above, in accordance with the Business Source License, use of
+// this software will be governed by the GNU Lesser General Public License v3.
+
+package processor_test_utils
```

Comment:
nit: since it is already inside processor folder, just `tests_utils` suffices, I think